assets/fonts/lib/fonts/dompdf_font_family_cache.dist.php

```diff
@@ -0,0 +1,107 @@
+<?php
+$fontDir = trailingslashit( WPO_WCPDF()->plugin_path() ) . 'assets' . DIRECTORY_SEPARATOR . 'fonts';
```

Comment:
I would follow the dompdf naming and use `$distFontDir` instead, this prevents mixups
https://github.com/dompdf/dompdf/blob/8768448244967a46d6e67b891d30878e0e15d25c/lib/fonts/dompdf_font_family_cache.dist.php#L2